spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-local.adoc

```diff
@@ -133,6 +133,14 @@ $ docker exec -it dataflow-server java -jar shell.jar
 
 Use of Spring Cloud Data Flow Shell is further described in <<shell,Shell>>.
 
+==== Spring Cloud Data Monitoring
```

Comment:
should `Spring Cloud Data Monitoring` be `Spring Cloud Data Flow Monitoring`?

Reply:
+1